src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.ConditionalAccessOperationTracker.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis
+{
+    internal sealed partial class ControlFlowGraphBuilder
+    {
+        private readonly struct ConditionalAccessOperationTracker
+        {
+            public readonly ArrayBuilder<IOperation>? Operations;
```

Comment:
>public readonly ArrayBuilder<IOperation>? Operations; [](start = 12, length = 53)

Consider adding a comment what these operations are and how they are related to conditional access.